Easydict/objc/ViewController/Window/WindowManager/EZWindowManager.m

```diff
@@ -455,6 +455,12 @@ - (void)updateFloatingWindowType:(EZWindowType)floatingWindowType isShowing:(BOO
 //    NSLog(@"after floatingWindowTypeArray: %@", self.floatingWindowTypeArray);
 }
 
+- (void)updateWindowsTitlebar {
+    [self.mainWindow.titleBar updateButtonsToolTip];
```

Comment:
```suggestion
    [_mainWindow.titleBar updateButtonsToolTip];
```

Normally, we should use `_mainWindow` because the main window is turn off by default, so there's no need to create it.